migrations/sql/V2019.01.16.11.23__alter_mine_party_appt.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE mine_party_appt
+ADD COLUMN processed_by character varying(60) DEFAULT '' NOT NULL,
```

Comment:
I don't like defaulting here, personally I'd rather see null if we don't know what the value is, because that's what null represents, but it's definitely easier to deal with if we know it's always a string. Not requesting a change, but it stands out to me. I see we've done this on the mine table for a few columns.

Reply:
I tend to agree with Ryan. It's far cleaner to have a NULL value that a default space. It could make things tricky for someone running a query on the data... if it looks like a NULL but isn't.

Could you also maybe add a comment for the new column? We're going to eventually clean up the table/column comments to make sure we have them all defined... so best to start now on anything new :)

Reply:
Kristen, maybe we can talk about this another time but I'd like to push back on column comments, unless there is some real exception in design. They are somewhat time consuming and don't tend to get maintained.

Reply:
yep sure thing :) I didn't know they were time consuming. Good to know. Thanks for the heads up.

Reply:
After some face to face discussion, "processed" probably isn't actually that intuitive. If a comment helps make it clear, I'm not opposed 👍

Reply:
Will happily change that to nullable if that's what we want to do. I wrote this early last week, so I don't remember exactly what I looked at, but this must have fallen in line with some sort of pattern that I saw.